cli/args/flags.rs

```diff
@@ -210,9 +210,22 @@ pub struct ReplFlags {
 pub struct RunFlags {
   pub script: String,
   pub watch: Option<WatchFlagsWithPaths>,
+  /// Internal. Sets the npm binary command name.
+  ///
+  /// This is only used for launching binary commands via `deno task` with byonm.
+  pub npm_bin_command_name: Option<String>,
```

Comment:
I was debating whether this should be a flag or an env var and ended up going with this (this arg is hidden). Not sure it's the right decision. I need to be able to tell Deno what the npm binary command name is.

Reply:
I don't have a strong preference between env var and flag - maybe we should err on the side that produces fewer lines of code? Given that there's >500 lines in this file maybe go with an env var?